doc/engineering-reference/src/surface-heat-balance-manager-processes/surface-heat-balance-with-moveable-insulation.tex

```diff
@@ -182,111 +182,90 @@ \subsubsection{Case8: Y\(_{0}\) not small, detailed conductance, with movable in
 \label{eq:HeatBalanceEquationCase8}
 \end{equation}
 
-\subsection{Fortran Algorithm Examples}\label{fortran-algorithm-examples}
+\subsection{C++ Algorithm Examples}\label{c++-algorithm-examples}
 
-While EnergyPlus has been translated from FORTRAN to C++, these code examples show the implementation of these different cases.  In most cases, variable names have not changed.  For more details, please the current EnergyPlus code.
+These C++ code examples, which are translated from the EnergyPlus FORTRAN code, show the implementation of these different cases.  In most cases, variable names have not changed.  For more details, please the current EnergyPlus code.
 
```

Comment:
"translated from the EnergyPlus FORTRAN code" --> "translated from the original EnergyPlus FORTRAN code".
"please the current EnergyPlus code" --> "please see the current EnergyPlus code"
Could also make that a link to the github repo.

OR, there's not that many lines here, maybe just copy the current code snippets and just say "These code example show the implementation of these different cases."

Reply:
The code snippets were taken from the current C++ implementation and are upto date but Case 5 and Case 6 do not exist in the code. They must have been merged with other cases by generalizing the code. That is what it looks now.  I have used the second option.